frontend/src/components/Banner.test.tsx

```diff
@@ -52,8 +57,23 @@ describe('Banner', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  it('does not show "Refresh" button if mode is "info"', () => {
+    const tree = shallow(
+      <Banner
+        message={'Some message'}
+        mode={'info'}
+        refresh={() => {
+          /* do nothing */
+        }}
+      />,
+    );
+    expect(tree.findWhere(el => el.text() === 'Refresh')).toEqual({});
```

Comment:
nit: I don't quite understand, why `toEqual({})`?
Let me take a look.

Reply:
Verified, `toEqual({})` will succeed no matter nodes found or not, because the wrapper helper object has no direct properties all the time.

Please use this way to verify not found:
https://github.com/kubeflow/pipelines/blob/fcd2559b2c846c2bd1e818cd72ebd8d64390258c/frontend/src/components/viewers/Tensorboard.test.tsx#L323-L330

Reply:
oh thanks for that, sorry I'm quite new to ts tests

Reply:
No worries, you are doing great